src/main/scala/gitbucket/core/view/helpers.scala

```diff
@@ -412,4 +412,10 @@ object helpers extends AvatarImageProvider with LinkConverter with RequestCache
    */
   def readableSize(size: Option[Long]): String = FileUtil.readableSize(size.getOrElse(0))
 
+
+  /**
+    * returns selected skin-name
+    */
+  def skinName(implicit context: Context): String = context.settings.skinName
```

Comment:
I think this is overkill. Writing `context.settings.skinName` directly in templates is enough in this situation instead of adding a helper method.